onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_api_helper.cc

```diff
@@ -25,20 +25,22 @@ static int32_t GetNNAPIRuntimeFeatureLevel(const NnApi& nnapi_handle) {
 }
 
 /**
- * Get the max feature level supported by all target devices.
+ * Get the max feature level supported by target devices. We want to run as more ops as possible
+  on NNAPI devices except nnapi-cpu. So we get the max feature level across target devices.
+
  *
  * @param nnapi_handle nnapi-lib handle.
  * @param device_handles target devices users want to use.
  *
- * @return The max feature level support by a set of devices.
+ * @return The max feature level support by a branch of devices.
```

Comment:
the meaning of "branch" is not so clear to me. how about:
```suggestion
 * @return The max feature level across all devices or the runtime feature level if no devices are specified.
```